src/main/scala/cromwell/engine/package.scala

```diff
@@ -10,5 +12,10 @@ package cromwell
  */
 
 package object engine {
+  type WorkflowId = UUID
 
+  sealed trait WorkflowState
+  case object WorkflowRunning extends WorkflowState
+  case object WorkflowFailed extends WorkflowState
+  case object WorkflowSucceeded extends WorkflowState
```

Comment:
Just FYI I find myself needing something like this for `Call`s.  It's not clear that these would be the same states as for workflows, but they might be.